src/os/bluestore/KernelDevice.cc

```diff
@@ -510,7 +510,7 @@ void KernelDevice::aio_submit(IOContext *ioc)
   // move these aside, and get our end iterator position now, as the
   // aios might complete as soon as they are submitted and queue more
   // wal aio's.
-  list<aio_t>::iterator e = ioc->running_aios.begin();
+  list<aio_t>::iterator e = ioc->running_aios.end();
```

Comment:
nope, should be begin().. note the loop on line 524, which should cover only the new ios

Reply:
OK, Thank you.

Reply:
Thanks!